hummingbot/strategy/pure_market_making/pure_market_making_v2.pyx

```diff
@@ -201,6 +207,9 @@ cdef class PureMarketMakingStrategyV2(StrategyBase):
     def cancel_order(self, market_info: MarketSymbolPair, order_id:str):
         return self.c_cancel_order(market_info, order_id)
 
+    def stop_order_tracking(self, market_info:MarketSymbolPair, order_id:str):
+        return self._sb_order_tracker.c_stop_tracking_limit_order(market_info, order_id)
```

Comment:
This is just the same as what the base class is doing, you don't need to define it again.